nmproxy/router/nftables_linux.go

```diff
@@ -92,6 +92,26 @@ var (
 			table: defaultIpTable,
 			chain: netmakerFilterChain,
 		},
+		{
+			nfRule: &nftables.Rule{
+				Table: filterTable,
+				Chain: &nftables.Chain{Name: "forward"},
```

Comment:
shouldn't this be `iptableFWDChain` instead of "forward" chain?

Reply:
yes